clients/cli/src/utils/prover.rs

```diff
@@ -1,22 +1,10 @@
-use std::collections::hash_map::DefaultHasher;
-use std::hash::{Hash, Hasher};
-
-// Distribution percentage (0.0 to 1.0) for the cancer-diagnostic program
-// Example: 0.01 = 1%, 0.5 = 50%, 1.0 = 100%
-const CANCER_DIAGNOSTIC_PERCENTAGE: f32 = 0.01;
+use crate::utils::experiment::Experiment;
 
 pub fn get_program_for_prover(prover_id: &str) -> String {
-    // Create a deterministic hash from the prover_id
-    let mut hasher = DefaultHasher::new();
-    prover_id.hash(&mut hasher);
-    let deterministic_hash = hasher.finish();
-
-    // Convert percentage (e.g., 0.01) to a number between 0-99
-    // Example: 0.01 -> 1, 0.5 -> 50
-    let percentage_of_cancer_program_in_programs = (CANCER_DIAGNOSTIC_PERCENTAGE * 100.0) as u64;
-
-    // If hash mod 100 falls below percentage, select cancer-diagnostic
-    let program_name = if deterministic_hash % 100 < percentage_of_cancer_program_in_programs {
+    // Ensure md5 is being used
+    debug_assert!(!Experiment::CANCER_DIAGNOSTIC.is_enrolled("52"));
+    debug_assert!(Experiment::CANCER_DIAGNOSTIC.is_enrolled("53"));
```

Comment:
Reading this on my phone, but 52 and 53 seem like magic numbers. Can you add some context please?

Reply:
This is doing the same thing as the unit test on web

Reply:
It's in a debug assertion, so I was trying to add code outside the assertion - though I guess if it's a constant maybe it's fine.

Reply:
Added comments in https://github.com/nexus-xyz/network-api/pull/83